exercises/grep/grep_test.exs

```diff
@@ -0,0 +1,259 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("bob.exs", __DIR__)
```

Comment:
I think we're loading the wrong file here - should probably be `grep.exs`, right?